src/DataSets/DataSets.component.js

```diff
@@ -128,6 +132,25 @@ const DataSets = React.createClass({
         });
     },
 
+    getLogs(datasets) {
+        // Set this.state.logs to the logs that include any of the given datasets.
+        if (datasets === null) {
+            this.setState({logsObject: null});
+        } else {
+            this.context.d2.dataStore.get('dataset-configuration').then(store => {
+                return store.get('logs').catch(() => []);
+            }).then(logs => {
+                const idsSelected = new Set(datasets.map(ds => ds.id));
+                const hasIds = (log) => log.datasets.some(ds => idsSelected.has(ds.id));
+                const logsSelected = logs.filter(hasIds);
+                logsSelected.sort(dateSort);
```

Comment:
Having lodash at hand, maybe: `const logsSelected = _(logs).filter(hasIds).sortBy("data").value();`

Reply:
Thanks for the tip! The problem is, if I understand correctly, that it'd be
```
  ... .sortBy("date").value()
```
and because `date` is a string, it would not sort correctly.

The function dateSort already converts that string into an actual date so the comparison is correct.

I also have a small preference for vanilla javascript if it doesn't result in any significant performance drop or less readability (and happy to fix it if that's the case!).

Reply:
Ah, yes, typo: `date`. 

You are correct, with the current format, it wouldn't work. This should work, though:  `sortBy(ds => new Date(ds.date))`

Usually dates are saved with ISO-8601 format, which is still a string, but the lexicographical comparison works. `const date = new Date().toISOString()`. However, this forces a parsing and reparsing to show a more human value, no need to do it here.

Lodash uses vanilla JS under the hood, there should be no performance hit.